tests/s2n_test.h

```diff
@@ -153,3 +153,13 @@ int test_count;
 
 /* Ensures fuzz test input length is greater than or equal to the minimum needed for the test */
 #define S2N_FUZZ_ENSURE_MIN_LEN( len , min ) do {if ( (len) < (min) ) return 0;} while (0)
+
+#define EXPECT_MEMCPY_SUCCESS(d, s, n)                                         \
```

Comment:
Does the check in [s2n_safety](https://github.com/awslabs/s2n/blob/master/tests/unit/s2n_safety_test.c#L126) need to be included?

Reply:
No, that is an exception as that is the test for the  `memcpy_check()` itself.